client/src/components/DataFiles/DataFilesListing/DataFilesListing.test.js

```diff
@@ -241,6 +241,36 @@ describe('DataFilesListing', () => {
     fireEvent.change(dropdownSelector, { target: { value: 'Folders' } });
     expect(getAllByTestId('file-listing-item').length).toBe(4);
     fireEvent.change(dropdownSelector, { target: { value: 'All Types' } });
-    expect(getAllByTestId('file-listing-item').length).toBe(filesFixture.listing.FilesListing.length);
-  })
+    expect(getAllByTestId('file-listing-item').length).toBe(
+      filesFixture.listing.FilesListing.length
+    );
+  });
+
+  it('does not render the Shared Workspaces component', () => {
```

Comment:
nice test 👍 
maybe rename to something more like,
```suggestion
  it('does not render the DataFilesSearchbar in the Shared Workspaces component when hideSearchBar is true', () => {
```